pages/k8s/1.18/components.md

```diff
@@ -329,34 +331,21 @@ These charms are frequently used with Charmed Kubernetes.
 These are the container images used by this release:
 
 <!-- GENERATED CONTAINER IMAGES -->
--   addon-resizer-amd64:1.8.5
--   addon-resizer-amd64:1.8.5
--   addon-resizer-arm64:1.8.5
--   addon-resizer-arm64:1.8.5
--   addon-resizer-s390x:1.8.5
--   addon-resizer-s390x:1.8.5
+-   cdk/addon-resizer-amd64:1.8.5  
 -   cephcsi/cephfsplugin:v1.0.0
 -   cephcsi/rbdplugin:v1.0.0
--   coredns/coredns-amd64:1.6.7
--   coredns/coredns-arm64:1.6.7
--   coredns/coredns-s390x:1.6.7
+-   cinder-csi-plugin:v1.17.0
+-   coredns/coredns:1.6.7
 -   coreos/kube-state-metrics:v1.9.5
+-   dashboard-amd64:v2.0.0-rc5
```

Comment:
Why remove the `kubernetesui/dashboard-x` prefix? Also, we do still use the `-$arch` variants, so I think the `[arm64|s390x]` variants should be kept in this list.